ngraph/src/ngraph/frontend/onnx_import/op/topk.cpp

```diff
@@ -36,23 +36,23 @@ namespace
         std::int64_t axis{node.get_attribute_value<std::int64_t>("axis", -1)};
 
         const auto data = node.get_ng_inputs().at(0);
-        const auto data_rank = data->get_output_partial_shape(0).rank();
+        const auto data_rank = data.get_partial_shape().rank();
         return ngraph::normalize_axis(node.get_description(), axis, data_rank);
     }
 
     /// \return Return the second input to the TopK node reshaped to a scalar.
-    std::shared_ptr<ngraph::Node> get_k(const ngraph::onnx_import::Node& node)
+    ngraph::Output<ngraph::Node> get_k(const ngraph::onnx_import::Node& node)
     {
         auto k_node = node.get_ng_inputs().at(1);
-        NGRAPH_CHECK(shape_size(k_node->get_shape()) == 1,
+        NGRAPH_CHECK(shape_size(k_node.get_shape()) == 1,
                      "ONNX TopK operator: 'K' parameter must contain a single positive value.",
                      node);
 
         return ngraph::onnx_import::reshape::interpret_as_scalar(k_node);
     }
 
     /// \return Return the outputs of the TopK node.
-    ngraph::NodeVector get_outputs(const std::shared_ptr<ngraph::Node>& node)
+    ngraph::OutputVector get_outputs(const std::shared_ptr<ngraph::Node>& node)
     {
         std::shared_ptr<ngraph::Node> values =
             std::make_shared<ngraph::op::v0::GetOutputElement>(node, 0);
```

Comment:
I guess you don't need GOE if you use OutputVector. Can you check that all works without GOE?

Reply:
BTW, agreed with @tsocha looks like we can remove this method.

Reply:
Removed